### PR TITLE
add minimal deployment configuration

### PR DESCRIPTION
**Pull request by aaperis**
### Describe the pull request:

- [x] Documentation change

### Pull request long description:

This is an attempt to cover the minimal set of configuration variables needed for a working deployment of the `SDA services`, `RabbitMQ` and `SDA Database` using the Helm charts. 
Changes may be needed when https://github.com/neicnordic/sensitive-data-archive/pull/398 is merged - comments are welcome :-)

### CODE REVIEW

File: docs/guides/deploy-k8s.md [hunk 1]
```diff
@@ -27,6 +27,118 @@ This guide explains how to deploy the Sensitive Data Archive (SDA) in kubernetes
 
 ## Minimal working configuration
 
+## Minimal set of configuration variables
+
+Here we provide a minimal list of variables that need to be configured in the `values.yml` file of the Helm charts in order to achieve a working deployment of the `sensitive data archive` services. In the following it is assumed that a *federated* setup is being deployed.
+
+## Global Variables
```
Comment (blankdots) [inline docs/guides/deploy-k8s.md:34]:
```suggestion
### Global Variables
```

File: docs/guides/deploy-k8s.md [hunk 2]
```diff
@@ -27,6 +27,118 @@ This guide explains how to deploy the Sensitive Data Archive (SDA) in kubernetes
 
 ## Minimal working configuration
 
+## Minimal set of configuration variables
+
+Here we provide a minimal list of variables that need to be configured in the `values.yml` file of the Helm charts in order to achieve a working deployment of the `sensitive data archive` services. In the following it is assumed that a *federated* setup is being deployed.
```
Comment (blankdots) [inline docs/guides/deploy-k8s.md:32]: maybe we can link to the files for each chart ? when the https://github.com/neicnordic/sensitive-data-archive/pull/398 is merged

File: docs/guides/deploy-k8s.md [hunk 3]
```diff
@@ -27,6 +27,118 @@ This guide explains how to deploy the Sensitive Data Archive (SDA) in kubernetes
...
+- `global.inbox.s3SecretKey`: The S3 secret key for the inbox.
+
+RabbitMQ:
+- `global.broker.host`: The host for the broker.
+- `global.broker.exchange`: The exchange for the broker.
+- `global.broker.routingError`: The routing error for the broker.
+- `global.broker.backupRoutingKey`: The backup routing key for the broker.
+
+Crypt4gh:
+- `global.c4gh.secretName`: The secret name for c4gh.
```
Comment (blankdots) [inline docs/guides/deploy-k8s.md:65]: maybe to specify that this is a k8s secret and it is the name by which it is referenced in the chart

File: docs/guides/deploy-k8s.md [hunk 4]
```diff
@@ -27,6 +27,118 @@ This guide explains how to deploy the Sensitive Data Archive (SDA) in kubernetes
...
+- `global.c4gh.keyFile`: The key file for c4gh.
```
Comment (blankdots) [inline docs/guides/deploy-k8s.md:66]: private key, if i remember correctly

File: docs/guides/deploy-k8s.md [hunk 5]
```diff
@@ -27,6 +27,118 @@ This guide explains how to deploy the Sensitive Data Archive (SDA) in kubernetes
...
+- `global.c4gh.passphrase`: The passphrase for c4gh.
+
+CEGA:
+- `global.cega.host`: The host for cega.
+- `global.cega.user`: The user for cega.
+- `global.cega.password`: The password for cega.
```
Comment (blankdots) [inline docs/guides/deploy-k8s.md:72]:
```suggestion
- `global.cega.host`: The host for [Federated EGA NSS API](https://nss.ega-archive.org/spec/#/).
- `global.cega.user`: The user for accessing Federated EGA NSS API .
- `global.cega.password`: The password for Federated EGA NSS API.
```

**Review by blankdots — COMMENTED** [review]
some suggestions also

File: docs/guides/deploy-k8s.md [hunk 6]
```diff
@@ -27,6 +27,118 @@ This guide explains how to deploy the Sensitive Data Archive (SDA) in kubernetes
...
 ## Minimal working configuration
 
+## Minimal set of configuration variables
+
+Here we provide a minimal list of variables that need to be configured in the [values.yml](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-svc/values.yaml) file of the Helm charts in order to achieve a working deployment of the `sensitive data archive` [services](https://github.com/neicnordic/sensitive-data-archive/tree/main/charts/sda-svc). In the following it is assumed that a *federated* setup is being deployed.
+
+### Global Variables
+
+TLS support:
+- `global.tls.issuer` or `glbal.tls.clusterIssuer`
```
Comment (dbampalikis) [inline docs/guides/deploy-k8s.md:37]:
```suggestion
- `global.tls.issuer` or `global.tls.clusterIssuer`
```

**Review by dbampalikis — APPROVED** [review]
It looks good in general. I am wondering though if we should have some examples for the values of the variables described here, since the `values.yaml` file doesn't provide them either. However, it might be an overkill and it is obvious for the users?

**Comment by pahatz** [comment]
It seems like we have a lot of outdated markdown files in the docs directory.
I think we're also missing direct instructions on how to deploy, those should be added too.

**Comment by aaperis** [comment]
> It seems like we have a lot of outdated markdown files in the docs directory. I think we're also missing direct instructions on how to deploy, those should be added too.

Goof point, this should be cleaned up as we go.

**Review by blankdots — APPROVED** [review]

**Pull request merged**